client/core/core.go

```diff
@@ -3301,7 +3313,19 @@ func (c *Core) connectDEX(acctInfo *db.AccountInfo) (*dexConnection, error) {
 			go c.handleConnectEvent(host, connected)
 		},
 		Logger: c.log.SubLogger(wsURL.String()),
-	})
+	}
+	if c.cfg.TorProxy != "" {
+		proxy := &socks.Proxy{
+			Addr:         c.cfg.TorProxy,
+			Username:     c.cfg.TorProxyUsername,
+			Password:     c.cfg.TorProxyPassword,
+			TorIsolation: true,
```

Comment:
Hmm, aren't user and pass ignored when TorIsolation=true?  https://github.com/decred/go-socks/blob/775b73941e73859df97ecc4f2f62952e754958a9/socks/dial.go#L120-L132

Reply:
Certainly looks that way.

Reply:
removed user/pass and made --torisolation